pallets/ah-migrator/src/staking/bags_list.rs

```diff
@@ -0,0 +1,72 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Fast unstake migration logic.
+
+use crate::*;
+use pallet_rc_migrator::staking::bags_list::alias;
+
+impl<T: Config> Pallet<T> {
+	pub fn do_receive_bags_list_messages(
+		messages: Vec<RcBagsListMessage<T>>,
+	) -> Result<(), Error<T>> {
+		let (mut good, mut bad) = (0, 0);
+		log::info!("Integrating {} BagsListMessages", messages.len());
```

Comment:
log target?

Reply:
Not sure if it adds much. Do you use it often to filter?